tools/cli/go-whisk/whisk/package.go

```diff
@@ -83,6 +85,39 @@ type PackageListOptions struct {
     Docs        bool                `url:"docs,omitempty"`
 }
 
+/*
+ *  Compare(s) compares Package p to Sortable s for the purpose of sorting.
+ *  params: Sortable type s that is also of type Package (REQUIRED)
+ *  ***Method of type Sortable***
+ *  ***By default, sorts Alphabetically***
+ */
+func(xPackage Package) Compare(sortable Sortable) bool{
+  // convert s back to proper type
```

Comment:
`s` doc here and above wasn't updated with variable name change. Looks like the same issue throughout each file this function is in.